apps/dashboard/src/utils/schema.ts

```diff
@@ -167,3 +167,31 @@ export const buildDefaultValues = (uiSchema: UiSchema): object => {
 
   return keys;
 };
+
+export const buildDefaultValuesOfDataSchema = (dataSchema: JSONSchemaDto) => {
```

Comment:
This function is a copy of the above one but the arg is just a different type. Can we adjust the above one?

Reply:
Modified it to handle nested defs